exonum/src/runtime/rust/mod.rs

```diff
@@ -12,7 +12,20 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! Built-in Rust runtime module.
+//! The basic runtime with the native Rust services.
+//!
+//! This runtime is usually presents in every blockchain instance and can only process a static set
+//! of available Rust services. To change this set you have to recompile the node binary.
```

Comment:
```suggestion
//! of available Rust services. This set is defined at the compile-time level. Once created, it can be changed only by the node binary recompilation.
```